docs/source/routing/performance/caching/response-caching/storage.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Response caching Redis storage for the GraphOS Router
+description: Response caching Redis storage for GraphOS Router with GraphOS Enterprise.
+minVersion: Router v2.8.0
+releaseStage: preview
+---
+
+
+## Redis configuration
```

Comment:
A lot of this configuration is duplicated in https://www.apollographql.com/docs/deploy-preview/4bf4ef1f4a66ddd9379bfb08/graphos/routing/performance/caching/distributed 

How do we makes sure these two pages are always in sync? Can we combine them and have one page for Redis config in the router? I do understand that there are some response caching specific config options too.

Reply:
i mostly tried to combine them.  but we're going to have to come back with a bigger docs refactor.